src/components/NavBar/index.tsx

```diff
@@ -5,14 +5,18 @@ import { HiOutlineShoppingBag } from "react-icons/hi";
 import { IoIosArrowBack } from "react-icons/io";
 import { RiArrowGoBackLine, RiArrowGoForwardLine } from "react-icons/ri";
 import { BiStar, BiPencil } from "react-icons/bi";
+import Link from "next/link";
+import HOME_PAGE_LINK from "@/constants/homePage";
 
 const NavigationBar = () => {
   return (
     <Grid templateColumns="repeat(3, 1fr)" bg="brand.primary" p={4}>
       <Flex alignItems="center">
-        <Button leftIcon={<IoIosArrowBack />} pl="0" variant="navbarIconBtn">
-          Home
-        </Button>
+        <Link href={HOME_PAGE_LINK} passHref>
+          <Button leftIcon={<IoIosArrowBack />} pl="0" variant="navbarIconBtn">
```

Comment:
Maybe consider using padding x as 8px and try if it looks better. Cos this padding left makes the home button weird on itss event shape. Or do you have a firm reason for using pl 0?
![image](https://user-images.githubusercontent.com/12379895/168465696-f90006e7-a7ed-4187-ae89-a34c5b40d8b7.png)